src/vmm/src/lib.rs

```diff
@@ -597,38 +597,6 @@ impl Vmm {
         Ok(vcpu_states)
     }
 
-    /// Restores vcpus kvm states.
-    pub fn restore_vcpu_states(
-        &mut self,
-        mut vcpu_states: Vec<VcpuState>,
-    ) -> Result<(), RestoreVcpusError> {
-        if vcpu_states.len() != self.vcpus_handles.len() {
-            return Err(RestoreVcpusError::InvalidInput);
```

Comment:
It seems that `RestoreVcpusError::InvalidInput` is no longer used, so we may be able to delete it.

Reply:
removed